packages/medusa/src/api/middlewares/normalized-query.ts

```diff
@@ -0,0 +1,20 @@
+import { NextFunction, Request, Response } from "express"
+
```

Comment:
suggestion (non-blocking, if-minor): Not really related to the PR @adrien2p so feel free to ignore, but I think it'd be cool to have a short comment here describing what this middleware does, wdyt?

Reply:
maybe something like this: 
```typescript
/**
 * Middleware for parsing an array like query param to an array
 * e.g: /admin/orders/?fields[]=id,status,cart_id becomes { fields: ["id", "status", "cart_id"] }
 */
```

Let's ask @olivermrbl to validate the intent 😄

Reply:
Correct me if I am wrong @adrien2p, but #1593 would still make use of `normalized-query`, so I believe adding a comment is reasonable no?

Reply:
👍 ill do that it is fine for me